plotly/statistical_charts.py

```diff
@@ -0,0 +1,263 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "marimo",
+#     "numpy==2.2.5",
+#     "pandas==2.2.3",
+#     "plotly==6.0.1",
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.13.6"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    import pandas as pd
+    import numpy as np
+    import plotly.express as px
+    import plotly.graph_objects as go
+    return go, mo, np, px
+
+
+@app.cell
```

Comment:
```suggestion
@app.cell(hide_code=True)
```
Recommend hiding markdown cell blocks for better visual appeal throughout the notebook.